packages/standards/README.md

```diff
@@ -0,0 +1,7 @@
+# Jetpack Standards
+
+A set of standards you can use to start working on any Jetpack project.
+
+### Usage
+
+Add the package to your project as a dev dependency and run `composer install`.
```

Comment:
Should we explain what this will do precisely? As a developer I wouldn't expect for a composer package to copy files to my working directory, to be honest 😉